ci/scripts/install_coiled_runtime.sh

```diff
@@ -10,7 +10,7 @@ then
   cat $1
   mamba env update --file $1
 else
-  mamba install -c conda-forge coiled-runtime=$COILED_RUNTIME_VERSION
+  mamba install -c conda-forge coiled-runtime=$COILED_RUNTIME_VERSION dask-snowflake
```

Comment:
Should we ping the dask-snowflake version we are testing? I wonder if this will update/downgrade some of the other packages that are installed with the runtime


I think here we are also missing the case for upstream and latest? not sure what's the best way to tackle this.

Reply:
yeah i'll add a pin